### DISCUSSION TITLE
Xcode 11.2.1 Support

### DISCUSSION THREAD

**Discussion by sinoru** [community/community#25552] [Actions]
<p>Hello, I’m really fan of GitHub actions which is greate.</p>
<p>Except… Recently Xcode 11.2 is deprecated by Apple, which required to use Xcode 11.2.1 for latest environment.</p>
<p>But, GitHub Actions workflow’s macOS VM doesn’t have Xcode 11.2.1. Which I need to download it in my workflow which eats a lot of minute.</p>
<p>I hope soon adds Xcode 11.2.1 on macOS VM. Thanks.</p>

**Comment by blane1988** [comment]
<p>I finished a long build, and got excited to see it attempt to deploy to TestFlight. Then I got an error saying it couldn’t deploy, due to this issue. I hope Xcode 11.2.1 gets added to Actions soon. Can any Actions devs give a possible timeframe? Thanks.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>An update to the mac image with Xcode 11.2, 11.2.1, and 11.3 beta1 will start rolling out this week! </p>

**Comment by kemalenver** [comment]
<p>Thanks for the update.  I’m waiting on this too.  Is there a way to get the right version of xcode any other way while we wait for the updated image?  I can foressee this happening a lot!</p>

**Comment by sinoru** [comment]
<p>You can use <code>xcversion</code> for download it in ci.</p>

**Comment by kevinabrioux** [comment]
<p>Could be interesting to update the doc :D </p>
<p><a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#macos-1015" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#macos-1015</a></p>

**Comment by colinhumber** [comment]
<p><a class="mention" href="/yanjingzhu">@yanjingzhu</a> Just a note, Xcode 11.2 has been deprecated by Apple and can’t be used for any deployments, so this version should probably be deprecated on here as well.</p>

**Comment by Kerizer** [comment]
<p>It’s more than a week past already and those your users, who are required to use swift 5.1.2 can not use your service as it has broken software. Any updates on this?</p>

**Comment by sinoru** [comment]
<p>I just create actions for select xcode if locally available or download from Apple Developer.</p>
<p><a href="https://github.com/marketplace/actions/setup-xcode" rel="nofollow noopener">https://github.com/marketplace/actions/setup-xcode</a></p>
<p>Maybe this helps until GitHub releases new VM Images for GitHub Actions Workflow.</p>

**Comment by blane1988** [comment]
<p>Hi Yanjingzhu,</p>
<p>Any update on when Xcode 11.2.1 will be available?</p>
<p>Thanks,</p>
<p>Ben</p>

**Comment by colinhumber** [comment]
<p><a class="mention" href="/blane1988">@blane1988</a> I’m running a build right now with Xcode 11.2.1. The documentation hasn’t been updated yet, by the looks of it.</p>
<p><a class="mention" href="/yanjingzhu">@yanjingzhu</a> It would be great when the Xcode deploys have finished to have the documentation updated. Thanks!</p>
